static/css/static.css

```diff
@@ -1748,4 +1748,4 @@ body.interface-hebrew #visualGardenPage .filter-title .reset {
 }
 .phone-number{
   unicode-bidi: plaintext;
-}
\ No newline at end of file
+}
```

Comment:
It would be nice if all text files had a blank line before EOF so that PRs like mine don't trigger this.